client/src/components/ManageAccount/tests/ManageAccountModals.test.js

```diff
@@ -347,8 +347,79 @@ describe('Edit Required Information', () => {
     });
 
     expect(getByText(/Submit/)).toBeDefined();
+  });  
+ 
+  it('should pass multiple valid phone numbers', async () => {
+    const stateWithFields = {
+      ...testState,
+      fields: {
+        countries: [[230, 'United States']],
+        institutions: [[1, 'University of Texas at Austin']],
+        ethnicities: [['Decline', 'Decline to Identify']],
+        genders: [['Other', 'Other']],
+        professionalLevels: [['Other', 'Other']],
+        titles: [['Other User', 'Other User']]
+      }
+    };
+    const storeWithFields = mockStore({ profile: stateWithFields });
+    stateWithFields['data']['demographics']['phone'] = '321-234-2342'
+    var {getByText,queryByText} = render(
+      <Provider store={storeWithFields}>
+        <EditRequiredInformationModal />
+      </Provider>
+    );
+    const phonecase =  ['321-234-2342','321-234-2342']
```

Comment:
This is a test to check non-valid cases, right? That said, isn't `'321-234-2342'` a valid number?

Also `phonecase` doesn't seem to be used.
```suggestion
  it('should render invalid message', async () => {
    const stateWithFields = {
      ...testState,
      fields: {
        countries: [[230, 'United States']],
        institutions: [[1, 'University of Texas at Austin']],
        ethnicities: [['Decline', 'Decline to Identify']],
        genders: [['Other', 'Other']],
        professionalLevels: [['Other', 'Other']],
        titles: [['Other User', 'Other User']]
      }
    };
    const storeWithFields = mockStore({ profile: stateWithFields });
    stateWithFields['data']['demographics']['phone'] = '321-234-2342'
    var {getByText,queryByText} = render(
      <Provider store={storeWithFields}>
        <EditRequiredInformationModal />
      </Provider>
    );
```